Packs/AnomaliSecurityAnalyticsAlerts/Integrations/AnomaliSecurityAnalyticsAlerts/AnomaliSecurityAnalyticsAlerts.yml

```diff
@@ -0,0 +1,109 @@
+category: Analytics & SIEM
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: AnomaliSecurityAnalyticsAlerts
+  version: -1
+configuration:
+- defaultvalue: https://optic.threatstream.com
+  display: Server URL
+  name: url
+  required: true
+  type: 0
+  section: Connect
+- display: Username
+  displaypassword: API Key
+  name: credentials
+  required: true
+  type: 9
+  section: Connect
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+  section: Connect
+  advanced: true
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+  section: Connect
+  advanced: true
+description: "The Anomali Security Analytics pack enables users to interact with Anomali ThreatStream's XDR API to manage security alerts. It supports creating search jobs, monitoring their status, retrieving results, and updating alert statuses or comments, streamlining integration with Palo Alto XSOAR."
+display: AnomaliSecurityAnalysisAlerts
+name: AnomaliSecurityAnalysisAlerts
+script:
+  commands:
+  - name: anomali-security-analytics-alerts-search-job-create
+    description: "Create a new search job"
+    arguments:
+    - name: query
+      description: Search expression or keyword you're looking for in logs.
+      required: false
+    - name: source
+      description: Filters results by the log source or origin system.
+      required: false
+    - name: from
+      description: Timerange - start time. Default value is 1 day
```

Comment:
Time/date formats must add examples to the descriptions of the fields (i.e. ISO8601 as 2020-01-01T00:11:22Z)